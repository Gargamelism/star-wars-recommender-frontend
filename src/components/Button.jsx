import React from "react";

export default function ButtonLink({ value, onClick }) {
    
    return (
        <input type="button" value={value} onClick={onClick} className="w-1/2 rounded-lg text-center text-white bg-blue-500 hover:bg-blue-600 cursor-pointer border-black border-2" />
    );
}