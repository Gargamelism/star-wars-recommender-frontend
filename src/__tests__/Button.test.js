import renderer from "react-test-renderer";
import Button from "../components/Button";

test("renders Button", () => {
  const tree = renderer
    .create(<Button value="123" />)
    .toJSON();
    
    expect(tree).toMatchSnapshot();  
});
