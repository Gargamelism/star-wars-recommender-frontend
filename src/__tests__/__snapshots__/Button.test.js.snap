// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Button 1`] = `
<input
  className="w-1/2 rounded-lg text-center text-white bg-blue-500 hover:bg-blue-600 cursor-pointer border-black border-2"
  type="button"
  value="123"
/>
`;
